src/App.js

```diff
@@ -2,14 +2,13 @@ import React from "react"
 import { Route } from "react-router-dom"
 import "./App.css"
 import "antd/dist/antd.css"
-import Auth from "./pages/Auth"
+import Auth from "./pages/Auth/Auth"
 import Home from "./pages/Home"
-import ROUTER from "./router"
 
 const App = () => (
   <div className="App">
-    <Route exact path={["/", "/login", "/register"]} component={Auth} />
-    <Route exact path={`/${ROUTER.HOME}`} component={Home} />
+    <Route exact path={["/", "login", "/register"]} component={Auth} />
+    <Route exact path="/home" component={Home} />
```

Comment:
create constant for routes, and use it in all places, const ROUTES = {
   HOME: '/',
   LOGIN: '/login',
  ....
}